snuba/datasets/groups.py

```diff
@@ -30,8 +30,9 @@
 
 
 class JoinedStorage(ReadableStorage):
-    def __init__(self, join_structure: JoinClause) -> None:
+    def __init__(self, storage_key: str, join_structure: JoinClause) -> None:
         self.__structure = join_structure
+        super().__init__(storage_key)
```

Comment:
Does this actually type check? It seems to me the parent class takes in a Cluster.

Reply:
Was testing a couple of options. Ended up changing the parent class to take storage key.